### PR TITLE
Knative Install on OpenShift

### LINKED ISSUE #1340: Install on OpenShift(docs/install/Knative-with-OpenShift.md)
## Expected Behavior
https://knative.dev/docs/install/scripts/knative-with-openshift.sh works

## Actual Behavior
https://knative.dev/docs/install/scripts/knative-with-openshift.sh can not find:
https://storage.googleapis.com/knative-releases/serving/latest/istio.yaml and https://storage.googleapis.com/knative-releases/serving/latest/release-lite.yaml

## Steps to Reproduce the Problem

1. follow the doc 
2. execute the script to install knative on openshift

## Additional Info
**Additional context**
n/a

**Install information:**
 - Fedora 30, oc 3.11
 - Knative Version: 0.5

### PR DESCRIPTION
**Pull request by joaedwar**
Fixes #1340 

## Proposed Changes

- Remove install script
- Install latest Knative Serving on OpenShift 4 using the operator on OperatorHub
- Include Images for screen shots of procedure

### CODE REVIEW

File: docs/install/Knative-with-OpenShift.md [hunk 1]
```diff
@@ -5,219 +5,47 @@ weight: 15
...
-wget https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz -o oc.tar.gz
-tar xzvf openshift*tar.gz
-mv openshift-origin-client-tools-*/oc .
-mv openshift-origin-client-tools-*/kubectl .
-rm -rf openshift-origin-client-tools-*/
-```
+## Installing the Knative Serving Operator
 
-You will now have the `oc` (and `kubectl`) binaries in your local directory.
+1. Go to **Catalog > OperatorHub** in the OpenShift Web Console. A list of operators for OpenShift, provided by Red Hat as well as a community of partners and open-source projects is provided. Click on the **Knative Serving Operator** tile.
```
Comment (samodell) [inline docs/install/Knative-with-OpenShift.md]: Make "OpenShift Web Console" into a link to the appropriate place.

File: docs/install/Knative-with-OpenShift.md [hunk 2]
```diff
@@ -5,219 +5,47 @@ weight: 15
...
 
-This guide walks you through the installation of the latest version of
-[Knative Serving](https://github.com/knative/serving) on an
-[OpenShift](https://github.com/openshift/origin) using pre-built images and
-demonstrates creating and deploying an image of a sample "hello world" app onto
-the newly created Knative cluster.
+# Knative Install on OpenShift
+
+This guide walks you through the installation of the latest version of [Knative
+Serving](https://github.com/knative/serving) on [OpenShift](https://github.com/openshift/origin) by using the Knative Serving Operator. The operator is available on the OpenShift OperatorHub. After you install Knative Serving, create and deploy an image of a sample "hello world" app onto the newly created Knative cluster.
```
Comment (samodell) [inline docs/install/Knative-with-OpenShift.md]:
```suggestion
Serving](https://github.com/knative/serving) on [OpenShift](https://github.com/openshift/origin) by using the Knative Serving Operator. The operator is available on the OpenShift OperatorHub. After you install Knative Serving, this guide walks you through creating and deploying an image of a sample "Hello World" app onto the new Knative cluster.
```

File: docs/install/Knative-with-OpenShift.md [hunk 3]
```diff
@@ -5,219 +5,47 @@ weight: 15
...
-mv openshift-origin-client-tools-*/kubectl .
-rm -rf openshift-origin-client-tools-*/
-```
+## Installing the Knative Serving Operator
 
-You will now have the `oc` (and `kubectl`) binaries in your local directory.
+1. Go to **Catalog > OperatorHub** in the OpenShift Web Console. A list of operators for OpenShift, provided by Red Hat as well as a community of partners and open-source projects is provided. Click on the **Knative Serving Operator** tile.
 
-## Scripted cluster setup and installation
+> **NOTE:** Use the **Filter by Keyword** box to facilitate the search of the Knative Serving operator in the catalog.
```
Comment (samodell) [inline docs/install/Knative-with-OpenShift.md]:
```suggestion
> **NOTE:** Use the **Filter by Keyword** box to help you find the Knative Serving operator in the catalog.
```

File: docs/install/Knative-with-OpenShift.md [hunk 4]
```diff
@@ -5,219 +5,47 @@ weight: 15
...
 
-For Linux and Mac, you can optionally run a
-[script](./scripts/knative-with-openshift.sh) that automates the steps on this
-page.
+![KSO Tile](images/knative_serving_tile_highlighted.png)
 
-Once you have `oc` present on your machine and in your `PATH`, you can simply
-run [this script](./scripts/knative-with-openshift.sh); it will:
+2. A **Show Community Operator** dialog box will open. Click **Continue** to proceed.
```
Comment (samodell) [inline docs/install/Knative-with-OpenShift.md]: I know the automatic numbering is not always working out correctly on the website, but we do want to stick to using 1. for each numbered item in a list. At some point the automatic numbering will be fixed, and we won't have to go back and update a lot of static numbers.

**Comment by joaedwar** [comment]
@samodell Please have a look. Thank you

**Pull request merged**